dir.props

```diff
@@ -487,18 +487,10 @@
 
   <!-- Default Test platform to deploy the netstandard compiled tests to -->
   <PropertyGroup>
-    <TestTFM Condition="'$(TestTFM)'=='' and ('$(TargetGroup)'=='netstandard1.7' or '$(TargetGroup)'=='netcoreapp1.1')">netcoreapp1.1</TestTFM>
```

Comment:
This code was causing some of the Xml tests to fail, as I think it forced them to run against netcoreapp1.0. I had to set $(TestTFM) and $(TestNugetTargetMoniker) in the csproj to workaround this issue. I talked to @ericstj and he helped figure out this is a bug in dir.props and he suggested this fix for it.